### PR TITLE
PXP-493: [Wukong CLI] Fix the text alignment when listing build artifacts

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-493](https://mindvalley.atlassian.net/browse/PXP-493)

## What's Changed

- [x] Fix the build artifact selection layout to respect the content width
before
Screenshot 2023-03-09 at 5 08 31 PM
Screenshot 2023-03-09 at 5 08 40 PM

after
Screenshot 2023-03-09 at 5 08 04 PM
Screenshot 2023-03-09 at 5 07 27 PM

[PXP-493]: https://mindvalley.atlassian.net/browse/PXP-493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/70#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/70#pullrequestreview-1334334769" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
